website/docs/r/vapp_network.html.markdown

```diff
@@ -45,6 +46,31 @@ resource "vcd_vapp_network" "vappNet" {
     end_address   = "192.168.2.50"
   }
 }
+
+# IPv6 example
```

Comment:
As per the explanation above - this should be a "##" section of its own and then a new separate  hcl snippet  should be opened (I cannot put proper example suggestion because triple backticks "```" clash 


```suggestion
## Example usage (IPv6)
```